OXRS-SHA-StateMonitor-ESP32-FW.ino

```diff
@@ -59,7 +59,7 @@
 uint8_t g_mcps_found = 0;
 
 // temperature update interval timer
-uint32_t g_last_temp_update = -TEMP_UPDATE_INTERVAL;
+uint32_t g_last_temp_update = -TEMP_UPDATE_INTERVAL_MS;

```

Comment:
I think all constants should include their units for clarity - sorry for mixing this in with this PR but I saw it and couldn't help myself!